docs/api/rvdss.md

```diff
@@ -0,0 +1,196 @@
+| `hpiv2_positive_tests`         | Number of positive HPIV-2 laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                    |
+| `hpiv3_positive_tests`         | Number of positive HPIV-3 laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                    |
+| `hpiv4_positive_tests`         | Number of positive HPIV-4 laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                    |
+| `hpivother_positive_tests`     | Number of positive other HPIV laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                |
+| `hpiv_positive_tests`          | Number of positive HPIV laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                      |
+| `hpiv_pct_positive`            | Percentage of HPIV laboratory tests that are positive (for all subtypes) <br/> **Earliest Date Available:** 2013-08-31                      |
+| `adv_tests`                    | Number of Adenovirus (ADV) laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                   |
+| `adv_positive_tests`           | Number of positive ADV laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                       |
+| `adv_pct_positive`             | Percentage of ADV laboratory tests that are positive <br/> **Earliest Date Available:** 2013-08-31                                          |
+| `hmpv_tests`                   | Number of Human Metapneumovirus (hMPV)laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                        |
+| `hmpv_positive_tests`          | Number of positive hMPV laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                                      |
+| `hmpv_pct_positive`            | Percentage of hMPV laboratory tests that are positive <br/>**Earliest Date Available:** 2013-08-31                                          |
+| `evrv_tests`                   | Number of Enterovirus/Rhinovirus (EV/RV) laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                     |
+| `evrv_positive_tests`          | Number of positive EV/RV laboratory tests <br/>**Earliest Date Available:** 2013-08-31                                                      |
+| `evrv_pct_positive`            | Percentage of EV/RV laboratory tests that are positive <br/>**Earliest Date Available:** 2013-08-31                                         |
+| `hcov_tests`                   | Number of Human Coronavirus (HCoV) laboratory tests <br/> **Earliest Date Available:** 2013-08-31                                           |
```

Comment:
suggestion: please add a note/warning that this is not the same as COVID with a little more detail. The name similarity could cause confusion.